gce/startup-script.sh

```diff
@@ -32,20 +32,21 @@ service google-fluentd restart &
 apt-get update
 apt-get install -yq ca-certificates git build-essential supervisor
 
+
+# Fetch source code
+export HOME=/root
+git clone https://github.com/GoogleCloudPlatform/nodejs-getting-started.git /opt/app
+
 # Install nodejs
 mkdir /opt/nodejs
-curl https://nodejs.org/dist/v16.15.0/node-v16.15.0-linux-x64.tar.gz | tar xvzf - -C /opt/nodejs --strip-components=1
+curl https://nodejs.org/dist/v22.14.0/node-v22.14.0-linux-x64.tar.xz | tar xvfJ - -C /opt/nodejs --strip-components=1
 ln -s /opt/nodejs/bin/node /usr/bin/node
 ln -s /opt/nodejs/bin/npm /usr/bin/npm
 
-# Get the application source code from the Google Cloud Repository.
-# git requires $HOME and it's not set during the startup script.
 export HOME=/root
```

Comment:
issue(non-blocking): Redundant `export HOME=/root` statement
suggestion: Remove the second one.